docs/activities/events/index.html

```diff
@@ -28,12 +28,14 @@ <h2>W3C WoT – Events</h2>
 		<li>
 			<strong>Upcoming</strong>
 			<ul>
-			    <li><a href="https://iiwot24.nws.cs.unibo.it/">IEEE IIWOT 2024, January 6, 2024</a></li>
+				<li><a href="https://www.w3.org/WoT/IG/wiki/Wiki_for_WoT_Week_2024_planning">WoT Week, 25-29 November 2024</a></li>
+				<li><a href="https://www.w3.org/2024/09/TPAC/Overview.html">W3C TPAC 2024, 23–27 September 2024</a></li>
```

Comment:
Shall we flip the 2 entries, first things first ?

Reply:
ah good point, I don't know I just followed the chronological order they would have when they are all past events...

Reply:
I got confused a bit now, not sure what we were following :D For a first time visitor, the "sooner" event should indeed be on top.

Reply:
Great, I think we should merge *before* TPAC is over ;-)